Header header

# 位置信息
float64 x
float64 y
float64 z
float64 yaw

# 速度信息
float64 v_linear
float64 v_angular

# 方向向量
float64 dir_x
float64 dir_y
